src/olympia/addons/views.py

```diff
@@ -143,8 +143,9 @@ def extension_detail(request, addon):
 def _category_personas(qs, limit):
     def fetch_personas():
         return randslice(qs, limit=limit)
+    # TODO: .query_key comes from cache-machine, find replacement
```

Comment:
the same cache-machine we're removing very soon?

Reply:
Yes! I added the todo here, replacing query_key is or should be part of the follow-up removal PR then